test/IRGen/huge_c_type.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-swift-frontend -import-objc-header %S/Inputs/huge_c_type.h %s -disable-llvm-optzns -emit-ir | %FileCheck %s
+// Make sure that this does not crash during LLVM's ISel. It does not like huge
+// llvm::IntegerTypes.
+// RUN: %target-swift-frontend -import-objc-header %S/Inputs/huge_c_type.h %s -c
+
+// REQUIRES: OS=macosx || OS=ios
```

Comment:
What makes this test iOS/macOS specific?

Reply:
See the first failure. It wants some module imported for size_t.